layers/+themes/theming/README.org

```diff
@@ -28,6 +28,30 @@ of the following form:
            (face2 attributes...)
            ...)
    ...)
+   
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
```

Comment:
1. Should be a sub-heading of "Usage"
2. Not a good heading name. "Example" is enough, a reference to TheBB's source can be added as a comment in the example code.